reference/data/models/blood/continuousglucose.v1.yaml

```diff
@@ -12,6 +12,10 @@ allOf:
         $ref: './glucose/trend.v1.yaml'
       trendRate:
         $ref: './glucose/trendrate.v1.yaml'
+      sampleInterval:
+        $ref: './glucose/sampleinterval.v1.yaml'
+      backfilled:
+        $ref: './glucose/backfilled.v1.yaml'
     required:
```

Comment:
Shouldn't this already include `trend`? And in light of the text in the other file, `sampleInterval` as well?

Reply:
See my other comment.

Reply:
Updated PR accordingly.